frontend/src/components/auth/Login.tsx

```diff
@@ -78,6 +113,19 @@ const Login = () => {
             Log In
           </button>
           <ResetPassword email={email} />
+
+          <GoogleLogin
+            clientId="175399577852-6hll8ih9q1ljij8f50f9pf9t2va6u3a7.apps.googleusercontent.com"
+            buttonText="Google"
+            onSuccess={(response: GoogleSucces): void => {
+              if ("tokenId" in response) {
+                onGoogleLoginSuccess(response.tokenId);
+              } else {
+                console.log(response);
```

Comment:
nit: can you make the error console.logs window alerts instead? that way we'll follow existing login error handeling

Reply:
Under what conditions will tokenId be missing? GoogleLoginResponseOffline?

Reply:
Yup, not sure when that error is returned though, the way it works is weird. I think it currently has some funky behaviour in incognito modes that I can't quite figure out. Forgot to report that :/